### DISCUSSION TITLE
分享一个生成动态badge的工具

### DISCUSSION THREAD

**Discussion by xgdyp** [X-lab2017/open-wonderland#101] [1️⃣ 新鲜事]
今天因为一些奇怪的原因突然发现原来shields.io的badge是可以定制的，没有体验过的同学可以先体验下
- https://shields.io/
顺藤摸瓜后发现，有个项目叫做substas可以帮助我更方便的制作动态更新的badge，项目地址和使用说明如下：
- https://github.com/spencerwooo/Substats
- https://zhuanlan.zhihu.com/p/116758511
借助这些工具，我们可以拓展很多操作：例如，把活跃度指标做成badge展示到readme中...

**Comment by will-ww** [comment]
感觉不错，可以用来要呀 ~
